frontend/lib/store_widgets/detail/detail_content.dart

```diff
@@ -27,6 +27,11 @@ class DetailContent extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final address = acceptingStore.address;
+    final street = address.street;
+    final location = "${address.postalCode} ${address.location}";
```

Comment:
Fixed in https://github.com/digitalfabrik/ehrenamtskarte/pull/439/commits/c8c133a1bf7507ede908fd5ba00b8eef937f5238

Reply:
Did the API definition change? Even if it is non-nullable in the database now, then the API should still serve it as nullable. Is this a change which will break existing clients?

Reply:
What api definition do you mean? This will definitely not break anything that wasn't broken before since there was no check whether `postalCode` is null before either.

Reply:
The API definition between the backend and the client.

> The worst that could happen is that a `null` is part of a string shown to users.

But that is not good, isn't? :thinking: 
If the postalCode can be null (by the definition of dart types), then it should be handled.

Reply:
Will do, but actually this would not be part of this PR.